projects/story-decorator/src/StoryDecorator.ts

```diff
@@ -89,6 +89,7 @@ const reduceMotionProperties = css`
     --spectrum-animation-duration-2000: 0ms;
     --spectrum-animation-duration-4000: 0ms;
     --spectrum-coachmark-animation-indicator-ring-duration: 0ms;
+    --spectrum-animation-duration-6000: 0ms;
```

Comment:
Can you move this up one line?

Reply:
Checkin on this.

Reply:
Just checked. These all are addressed. Thanks